app/api/files/PDF.js

```diff
@@ -1,7 +1,7 @@
 import EventEmitter from 'events';
 import fs from 'fs';
 import path from 'path';
-import languages from 'shared/languages';
+import { detect } from 'shared/detectLanguage';
```

Comment:
this variable name is too generic, please change to detectLanguage or something similar ?

Reply:
changed